tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -237,8 +236,21 @@ sub install_kubevirt_packages {
     zypper_call('in jq open-iscsi') if (script_run('rpmquery jq open-iscsi') && ($kubevirt_ver ge "0.50.0"));
 
     # Install required packages perl-CPAN-Changes and ant-junit
+    my $repo_name = '';
+    my $inst_cmd = 'in -f';
```

Comment:
If these two variables ```repo_name``` and ```inst_cmd``` are only used inside if block, can they moved into there ?

Reply:
Thanks for review! I made relative changes. For the variable `repo_name`, it requires to generate random number for each repository name as per the code logic.